worldeditadditions/lib/selection/cloud.lua

```diff
@@ -4,8 +4,7 @@
 -- ██      ██      ██    ██ ██    ██ ██   ██
 --  ██████ ███████  ██████   ██████  ██████
 worldeditadditions.add_pos = {}
-local wea = worldeditadditions
-function worldeditadditions.add_point(name, pos)
+function worldeditadditions.selection.add_point(name, pos)
```

Comment:
You'll need to define `worldeditadditions.selection` like this if you haven't already:

```lua
worldeditadditions.selection = {}
```

Reply:
this is now outdated